biodatacatalyst-ui/src/main/webapp/psamaui/login/login.js

```diff
@@ -1,5 +1,7 @@
-define(['common/session', 'picSure/settings', 'common/searchParser', 'jquery', 'handlebars', 'text!login/login.hbs', 'text!login/not_authorized.hbs', 'overrides/login', 'util/notification', 'login/fence_login'],
-		function(session, settings, parseQueryString, $, HBS, loginTemplate, notAuthorizedTemplate, overrides, notification, fenceLogin){
+define(['common/session', 'psamaSettings/settings', 'common/searchParser', 'jquery', 'handlebars', 'text!login/login.hbs',
+        'text!login/not_authorized.hbs', 'psamaui/overrides/login', 'util/notification', 'login/fence_login'],
```

Comment:
This may not be a task for right now, but I would like to see the fence_login.js file (from BDC overrides) completely replace the login.js (from hpds-ui) instead of loading both and deciding which one to use dynamically.  It might be a bit tricky, so we can probably put this off, but maybe log it somewhere?